Example/Podfile.lock

```diff
@@ -61,7 +61,7 @@ PODS:
 DEPENDENCIES:
   - BlinkEReceipt (~> 2.0)
   - BlinkReceipt (~> 1.27)
-  - CaptureReceipt (from `https://github.com/tiki/capture-receipt-ios.git`)
+  - CaptureReceipt (from `../../capture-receipt-ios`)
```

Comment:
run pod install after updating the dependencies to update podspec.lock

Reply:
done